e2e/testenv/infra/cmd/setup.go

```diff
@@ -163,30 +166,14 @@ var setupCmd = &cobra.Command{
 				return "", nil
 			})
 
-			fmt.Println("determining Helm binary path...")
-			helmPath := os.Getenv("HELM_PATH")
-			if helmPath == "" {
-				helmPath = "/usr/bin/helm" // prevents eg. ~/.rd/bin/helm from being used, without support for skipping TLS
+			// TODO enable this when the Helm library supports `--insecure-skip-tls-verify`
```

Comment:
Should we get rid of this comment then, or have I missed something? :)

Reply:
as it is very possible that the `helmops` PR takes time to be merged... https://github.com/rancher/fleet/pull/3100